src/Microsoft.Health.SqlServer/Features/Client/SqlConnectionWrapper.cs

```diff
@@ -38,7 +38,7 @@ public class SqlConnectionWrapper : IDisposable
             }
             else
             {
-                SqlConnection = new SqlConnection(configuration.ConnectionString);
+                SqlConnection = SqlConnectionHelper.GetSqlConnection(configuration.ConnectionString, configuration.UseManagedIdentity);
```

Comment:
This won't work in the managed service because how access token is obtained is different in SF vs AKS vs here.

What we should do is have an interface that allows the initialization of the SqlConnection and then it can be overwritten by the IoC container for each specific service.

Reply:
Addressed in the new iteration.